<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="message">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessage">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Küldés</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Üzenet:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Státusz:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="trafficLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No traffic.</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabTypes">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>501</width>
      <height>361</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPopeye">
     <attribute name="title">
      <string>Popeye</string>
     </attribute>
     <widget class="QLabel" name="addressLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cím:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutChoose">
       <item>
        <widget class="QRadioButton" name="rb_GetMeasurements">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Feszültség, Áram, Teljesítmény, Hőmérséklet kiolvasása</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_LED_Check">
         <property name="text">
          <string>LED ellenőrzése</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_PID">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>PID kiolvasása</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableGetMeasurements">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>27</number>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Voltage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Current</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Power</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Temp.</string>
       </property>
      </column>
     </widget>
     <widget class="QTextEdit" name="textGetMeasurements">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBFU">
     <attribute name="title">
      <string>BFU</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rb_SerialNumber_2">
         <property name="text">
          <string>Sorozat szám kiolvasása</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_Voltage_2">
         <property name="text">
          <string>Feszültség érték kiolvasása</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_Current_2">
         <property name="text">
          <string>Áram érték kiolvasása</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_LED_Check_2">
         <property name="text">
          <string>LED ellenőrzése</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_EEPROM_2">
         <property name="text">
          <string>EEPROM kiolvasása</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBFURack">
     <attribute name="title">
      <string>BFU Rack</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="rb_SerialNumber_3">
         <property name="text">
          <string>Sorozat szám kiolvasása</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_Voltage_3">
         <property name="text">
          <string>Feszültség érték kiolvasása</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_Current_3">
         <property name="text">
          <string>Áram érték kiolvasása</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_LED_Check_3">
         <property name="text">
          <string>LED ellenőrzése</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_EEPROM_3">
         <property name="text">
          <string>EEPROM kiolvasása</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="copyright">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>www.moresoftwares.com       2018</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="serialPortLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Serial port:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serialPort"/>
     </item>
     <item>
      <widget class="QLabel" name="timeoutLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Válasz idő, msec:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeout"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>PID Data</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutPIDLabel">
      <item>
       <widget class="QLabel" name="label_ProductNumber">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Product Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_RevNumber">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Rev Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ProductName">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Product Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ProductionDate">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Production Date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SerialNumber">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Serial Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="text_ProductNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_RevNmber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_ProductName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_ProductionDate">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_SerialNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_LED">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Fault LED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fault LED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busy LED</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial_Communication">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuSerial_Communication"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
